include/stomp_moveit/trajectory_visualization.hpp

```diff
@@ -103,7 +103,7 @@ get_success_trajectory_publisher(rclcpp::Publisher<visualization_msgs::msg::Mark
 
       const moveit::core::LinkModel* ee_parent_link = group->getOnlyOneEndEffectorTip();
 
-      if (ee_parent_link != nullptr)
+      if (ee_parent_link != nullptr && !trajectory.empty())
```

Comment:
I was assuming that a successful trajectory wouldn't be empty.

Reply:
Is it guaranteed that it is not empty? This is just a sanity check